src/FIREGENTO/Magento/Command/Eav/RemoveUnusedMediaCommand.php

```diff
@@ -65,7 +65,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $directoryIterator = new \RecursiveDirectoryIterator($imageDir);
             foreach( new \RecursiveIteratorIterator($directoryIterator) as $file) {
 
-                if(strpos($file, "/cache") !== false || is_dir($file) ) {
+                if(strpos($file, "/cache") !== false || strpos($file, "/placeholder") !== false || is_dir($file) ) {
```

Comment:
issue with placeholder folder was resolved in https://github.com/magento-hackathon/EAVCleaner/commit/b011a04265a95a30c1e388c014d7a0b40627d0da